test/runner/requirements/units.txt

```diff
@@ -4,6 +4,7 @@ cryptography
 pycrypto
 jinja2
 mock
+mocker
```

Comment:
If this was a requirement for our unit tests then they would be failing already. What is it that uses this?

Reply:
I misinterpreted this error:

    E       fixture 'mocker' not found

And there was a package named 'mocker'.

On Python 2.7 I have lots of unit test failures, and the majority is because of the above error.

It seems ncclient is a dependency for netconf ?

Reply:
The mocker-errors are related to pytest-mock missing from the system.